tests/xdsl_opt/test_xdsl_opt.py

```diff
@@ -146,9 +146,7 @@ def test_print_between_passes():
         opt.run()
 
     output = f.getvalue()
-    assert (
-        len([l for l in output.split("\n") if "builtin.module" in l]) == len(passes) + 1
-    )
+    assert len([l for l in output.split("\n") if "builtin.module" in l]) == len(passes)
```

Comment:
What changed here?
Were we printing too much at one point?

Reply:
Yeah, the print-between-passes also printed after the last pass, which seems wrong to me